### PR TITLE
Fix accessibility issues page editing

### PR DESCRIPTION
**Pull request by chrispymm**
This PR improves the accessible labelling on all other edit screens.  

Bumps to presenter 3.3.33 in order to use the template changes made there.

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -58,6 +58,8 @@
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
+  config.hosts << 'b6a5-82-0-200-55.ngrok-free.app'
```
Comment (StevenLeighton21) [inline config/environments/development.rb:61]: Does this need to stay long term?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -96,42 +96,42 @@
...
-  enable_coverage(:branch)
-end
-
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new(
-  [
-    SimpleCov::Formatter::HTMLFormatter,
-    SimpleCov::Formatter::Console
-  ]
-)
+# SimpleCov.start do
```
Comment (StevenLeighton21) [inline spec/spec_helper.rb:99]: No more simplecov is a good thing in my book but I guess it's unintentional here? :D

File: config/environments/development.rb (hunk 1 above)
Comment (chrispymm) [reply config/environments/development.rb:61]: ah no, sorry, probably me doing an inadvertant commit all 
This was when I was trying to do screenreader testing through a VM, which couldn't access my local network, so was using ngrok as a proxy.
I'll remove it

File: spec/spec_helper.rb (hunk 2 above)
Comment (chrispymm) [reply spec/spec_helper.rb]: Haha - yeah I comment it out locally cos it means I can't see the actual test results!
This was proabbly also a result of my inadvertant commit all, but in this case I thought I'd pushed the fix before requesting review.  Apparently not!

**Review by StevenLeighton21 — APPROVED** [review]

**Review by H3ll3m4 — APPROVED** [review]
🚀

**Pull request merged**